repos.yaml

```diff
@@ -288,6 +288,12 @@
   channel: 'knative-functions'
   assignees: 'knative-extensions/func-tastic-approvers'
 
+- name: 'knative-extensions/func-go'
+  meta-organization: 'knative-extensions'
+  fork: 'knative-automation/func-go'
+  channel: 'knative-functions'
+  assignees: 'knative-extensions/function-runtime-approvers'
```

Comment:
Just noticed the wrong name of approve team, fixed in latest commit. Pls another lgtm is neeeded. :)